src/components/Dropzone/index.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { ChangeEvent, useRef } from 'react'
+import styled from '@emotion/styled'
+import { color, space, radius, fontWeight } from '../../theme'
+import { H4 } from '../Heading'
+import { SmallText } from '../Text'
+import { BaseButton } from '../BaseButton'
+import ILLUSTRATION from './files.svg'
+
+interface DropzoneProps {
+  title: string
+  subtitle: string
+  action: string
+  ariaLabel?: string
+  accept?: Array<string>
```

Comment:
`string[]` is what we use on most places

Reply:
oops, fixed!